src/main/java/com/iri/training/web/service/UserServiceImpl.java

```diff
@@ -2,32 +2,36 @@
 
 import java.sql.SQLException;
 
+import org.apache.log4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 import com.iri.training.model.User;
+import com.iri.training.repository.UserCommentRepositoryImpl;
 import com.iri.training.repository.UserRepository;
 
 @Service
 public  class UserServiceImpl implements UserService {
+	Logger logger = Logger.getLogger(UserCommentRepositoryImpl.class);
```

Comment:
Use the correct class in logger: Logger logger = Logger.getLogger(UserServiceImpl.class);